roofit/roostats/inc/RooStats/SimpleLikelihoodRatioTestStat.h

```diff
@@ -25,30 +25,27 @@ namespace RooStats {
 
    public:
 
-      //__________________________________
+      /// Constructor for proof. Do not use.
       SimpleLikelihoodRatioTestStat() :
          fNullPdf(NULL), fAltPdf(NULL)
       {
-         // Constructor for proof. Do not use.
          fFirstEval = true;
-        fDetailedOutputEnabled = false;
-        fDetailedOutput = NULL;
+         fDetailedOutputEnabled = false;
+         fDetailedOutput = NULL;
          fNullParameters = NULL;
          fAltParameters = NULL;
-    fReuseNll=kFALSE ;
-    fNllNull=NULL ;
-    fNllAlt=NULL ;
+         fReuseNll=kFALSE ;
+         fNllNull=NULL ;
+         fNllAlt=NULL ;
       }
 
-      //__________________________________
+      /// Takes null and alternate parameters from PDF. Can be overridden.
```

Comment:
/// Constructor passing PDF for the null and the alternate models. 
/// Take the corresponding parameters from the pdf's.